src/schema/slack/types/custom/message_context.ts

```diff
@@ -0,0 +1,21 @@
+import SchemaTypes from "../../../schema_types.ts";
+import { SlackPrimitiveTypes } from "../../types/mod.ts";
+import { DefineType } from "../../../../types/mod.ts";
+
+const MessageContextType = DefineType({
+  name: "slack#/types/message_context",
+  type: SchemaTypes.object,
+  properties: {
+    message_ts: {
+      type: SlackPrimitiveTypes.message_ts,
+    },
+    user_id: {
+      type: SlackPrimitiveTypes.user_id,
+    },
+    channel_id: {
+      type: SlackPrimitiveTypes.channel_id,
+    },
+  },
```

Comment:
We should add `required: ["message_ts"]` here since that always must be provided. The others are optional